node/resolvers/geolocation.ts

```diff
@@ -168,6 +171,12 @@ function getCountry(googleAddress: google.maps.GeocoderResult) {
     : null
 }
 
+declare var process: {
+  env: {
+    VTEX_APP_ID: string
+  }
+}
```

Comment:
can you move this to `node/typings/global.d.ts`?

Reply:
It seems this repository is missing node types. Could you run `yarn add --dev @types/node`? After that I don't think you'll need this anymore.

Reply:
@marcoskwkm I tried that, but it still requires my declaration... I think it makes sense since it's a custom environment variable.

@lucasecdb I've moved the declaration to `global.d.ts`. Also I changed a bit how it was declared... I did some research and I think this way is more standardized.

Reply:
Weird, I tested here and just by adding `@types/node` Typescript identifies `VTEX_APP_ID` as being `string | undefined`, but moving the declaration to `global.d.ts` is also fine.